modular_skyrat/modules/modular_vending/code/autodrobe.dm

```diff
@@ -37,6 +37,8 @@
 		/obj/item/clothing/head/cowboyhat/sheriff/alt = 5,
 		/obj/item/clothing/head/cowboyhat/deputy = 5,
 		/obj/item/clothing/shoes/jackboots/timbs = 1,
+		/obj/item/clothing/under/costume/loincloth/sensor = 5,
```

Comment:
ehhhh on these being in the autodrobe but that may be me

Reply:
it was in it beforehand, so